### ISSUE TITLE
Markdown cleanup: use {{% %}} rather than {{< >}}

### ISSUE THREAD

**Issue by chalin** [open-telemetry/opentelemetry.io#1149]

**Comment by bshramin** [comment]
@chalin Can you please assign this issue to me?

**Comment by chalin** [comment]
There's another issue that I need to resolve before work on this can begin. I'll post here once I have an update.

**Comment by svrnm** [comment]
@chalin is this one still relevant? I still see a lot of `{{<` in our pages.

**Comment by chalin** [comment]
Yes it is still relevant. I still haven't had time to address the (Docsy) issue I mentioned back in 2022 🤷🏼‍♂️. I am hopeful that I can get that fixed (it's related to the `alert` shortcode if I'm not mistaken) within the next two quarters 😄.

**Comment by chalin** [comment]
That being said, the cleanup can be started as long as we do not yet touch the `alert` shortcodes (well, technically, they can be touched, but it's not guaranteed to work -- so maybe just best to wait).

**Comment by heidi04aldidi** [comment]
@chalin Hi, I would like to work on this issue. Please let me know if that’s fine, and I’ll get started!
